.goreleaser.yml

```diff
@@ -80,6 +79,48 @@ nfpms:
       postinstall: ./docs/packaging/postinstall.sh
       postremove: ./docs/packaging/postremove.sh
 
+kos:
+  - id: ghcr
+    repository: ghcr.io/juanfont/headscale
+    base_image: gcr.io/distroless/base-debian11
+    build: headscale
+    main: ./cmd/headscale
+    env:
+      - CGO_ENABLED=0
+    platforms:
+      - linux/amd64
+      - linux/386
+      - linux/arm64
+      - linux/arm/v7
+      - linux/arm/v6
```

Comment:
arm/v6 and arm/v5 are hardly used and we didn't build images for them beforehand. I would keep it simple arm64 and arm/v7 these platform are more commonly used for example newer raspberry pies.